Source/WebCore/dom/Document.idl

```diff
@@ -87,7 +87,7 @@ typedef (HTMLScriptElement or SVGScriptElement) HTMLOrSVGScriptElement;
     attribute boolean xmlStandalone;
 
     // Non standard: It has been superseeded by caretPositionFromPoint which we do not implement yet.
-    Range caretRangeFromPoint(optional long x = 0, optional long y = 0);
+    [ImplementedAs=caretRangeFromPointForBindings] Range caretRangeFromPoint(optional long x = 0, optional long y = 0);
```

Comment:
I would prefer doing this the other way around here and elsewhere. At least I was taught that we generally aim for the C++ API to match the web standard API.

Reply:
Got it — as we discussed over Slack, we should actually be able to remove these `-ForBindings` variants altogether, and instead just have `caretRangeFromPoint(int x, int y, HitTestSource = HitTestSource::Script)`. The internal call site can just pass in `HitTestSource::User` directly.